tests/pausing_test.go

```diff
@@ -335,7 +335,7 @@ var _ = Describe("[rfe_id:3064][crit:medium][vendor:cnv-qe@redhat.com][level:com
 
 			})
 
-			It("[test_id:3083]should gracefully handle console connection", func() {
+			It("[test_id:3083]should connect to serial console", func() {
```

Comment:
I can't comment on the right line, I meant line 343.

You can also test that the warning message is indeed showing up.
Something like:
```
command := tests.NewVirtctlCommand("pause", "vm", "--namespace", tests.NamespaceTestDefault, vm.Name)
out := command.InOrStdin()
scanner := bufio.NewScanner(out)
scanner.Buffer(make([]byte, 1024), 512*1024)
for scanner.Scan() {
	// Ensure your message was printed
}
```

Reply:
The warning message is printed on virtctl console command, not on pause. I'm not sure it is feasible to automate virtctl console command.

Reply:
Oh, got you.
I'm not sure it's feasible too. Thanks!